src/js/hooks/cycleCount/useCountStepTable.jsx

```diff
@@ -158,10 +159,18 @@ const useCountStepTable = ({
             value={value}
             onChange={onChange}
             onBlur={onBlur}
-            className={`m-1 ${showTooltip ? 'w-99' : 'w-75'}`}
-            errorMessage={error}
+            className={`m-1 ${showTooltip ? 'w-99' : 'w-75'} ${error && 'border border-danger'}`}
+            hideErrorMessage
             {...fieldProps}
           />
+          {error && (
+            <TooltipWrapper
+              content={error}
+              className="error-icon"
+            >
+              <RiErrorWarningLine />
+            </TooltipWrapper>
+          )}
```

Comment:
1. I would change the name of this component, because TooltipWrapper, suggests that the Tooltip should be passed as children of this because it is wrapping the passed children.
2. We should add a property to pass a custom icon instead of placing it as a wrapped component. The content here should be wrapped.